### ISSUE TITLE
KITE demultiplexing fails on google colab 

### ISSUE THREAD

**Issue by dylanmr** [pachterlab/kb_python#55]
Thanks for the great tool, really enjoying working with kallisto!

**Describe the issue**

kb KITE mismatch index will not build on google colab or on my local machine

**What is the exact command that was run?**
```
kb ref -i mismatch.idx -f1 mismatch.fa -g t2g.txt --workflow kite features.tsv
```

```
An exception occurred
Traceback (most recent call last):
  File "/tools/anaconda/envs/lhv464/kallisto/lib/python3.7/site-packages/kb_python/main.py", line 670, in main
    COMMAND_TO_FUNCTION[args.command](parser, args, temp_dir=temp_dir)
  File "/tools/anaconda/envs/lhv464/kallisto/lib/python3.7/site-packages/kb_python/main.py", line 101, in parse_ref
    if len(args.fasta) != len(args.gtf):
TypeError: object of type 'NoneType' has no len()
[2020-01-24 17:29:39,842]   ERROR An exception occurred
```

**Comment by Lioscro** [comment]
Should be fixed with 0d9ac86ba3c787e09903308d3af426ff0b41ac07.
Could you try again and let us know?

**Comment by dylanmr** [comment]
works great now! thanks!
